src/main/java/org/gridsuite/loadflow/server/LoadFlowService.java

```diff
@@ -68,12 +71,25 @@ private Network getNetwork(UUID networkUuid) {
         }
     }
 
-    LoadFlowResult run(UUID networkUuid, String variantId, List<UUID> otherNetworksUuid, LoadFlowParameters parameters,
+    private LoadFlowParameters buildLoadFlowParameters(LoadFlowParametersInfos loadflowParams, String provider) {
+        LoadFlowParameters params = loadflowParams == null || loadflowParams.getCommonParameters() == null ?
+             new LoadFlowParameters() : loadflowParams.getCommonParameters();
```

Comment:
LoadFlowParameters.load() would be better than new LoadFlowParameters() to allows loading default values specific to some deployment